src/controllers/TextSelectionController.ts

```diff
@@ -0,0 +1,28 @@
+import { EditorAPI } from '../../types/CommonTypes';
+import { TextStyleUpdateType } from '../../types/TextStyleTypes';
+/**
+ * The TextSelectionController is responsible for all communication regarding text styles.
+ * Methods inside this controller can be called by `window.SDK.textStyle.{method-name}`
+ */
+export class TextSelectionController {
```

Comment:
this doesnt feel correct, what has text styles to do with text selection? naming feels off

Reply:
DDone 3b231437acc17ad1deb4db49d47128d9ebdca9a9